 this definition is private for binutils, seems the author forgot that
 when decide to use it in execstack :-)
 may be next bintuils will make it public.
  -Alejandro

--- ./src/prelink.h.orig	2003-12-09 22:59:17.000000000 -0300
+++ ./src/prelink.h	2003-12-09 23:04:03.000000000 -0300
@@ -44,6 +44,8 @@
 #define SHT_GNU_LIBLIST		0x6ffffff7
 #endif
 
+#define PT_GNU_STACK		(PT_LOOS + 0x474e551)
+
 struct prelink_entry;
 struct prelink_info;
 struct PLArch;
